docs/modules/ROOT/content-nav.adoc

```diff
@@ -25,6 +25,10 @@
 **** xref:subscriptions/events/create.adoc[Create]
 **** xref:subscriptions/events/update.adoc[Update]
 **** xref:subscriptions/events/delete.adoc[Delete]
+*** xref:subscriptions/scaling.adoc[]
+*** xref:subscriptions/plugins/index.adoc[Plugins]
```

Comment:
This does not unfold a submenu like "Events" does just above. Do you see that too?